config.js

```diff
@@ -389,11 +386,11 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
 
     if (argv.banunranked) {
         for (let i of argv.banunranked.split(',')) {
-            banned_unranked_users[i] = true;
+            exports.banned_unranked_users[i] = true;
         }
     }
 
-    if (argv.minrank) {
+    if (argv.minrank && !argv.minrankranked && !argv.minrankunranked) {
```

Comment:
I have some doubts about ignoring `minrank` if `minrankranked` is set. Why not just do both checks if the admin decide to have both set? Less code and arguably more flexible settings.

Reply:
because let's imagine bot admin is careless and does : 

- minrank : 15k
- minrankranked : 1d
- minrankunranked 15k

then all games will allow 15k+, while bot admin wanted 1d+ for ranked, and 15k+ for unranked

this adds reliability to gtp2ogs in case bot admin is a bit careless or not knowledgeable about gtp2ogs